tests/unit/main_html5/wrapper-api-tests.js

```diff
@@ -75,7 +75,36 @@ describe('main_html5 wrapper tests', function () {
     spyOn(element.seekable, "start").andReturn(0);
     spyOn(element.seekable, "end").andReturn(duration);
     element.seekable.length = 1;
-  }
+  };
+
+  var setDvr = function(start, end) {
+    start = start || 0;
+    end = end || 1750;
+
+    // This can be called multiple times in order to update the DVR window,
+    // we need to remove the spies before setting new values.
+    if (element.__dvrSpy) {
+      element.__dvrSpy.restore();
+    }
```

Comment:
Should this be called in a afterEach/beforeEach? Would have less of a chance of affecting tests that don't call setDvr.

Reply:
The spies are attached to the video element which is re-created before each unit test. Right now the restore is only needed when we want to update the DVR window times in the same unit test.

Reply:
Got it